test/src/unit-iterators2.cpp

```diff
@@ -307,6 +317,7 @@ TEST_CASE("iterators 2")
                     auto it = j_array.begin();
                     it += 3;
                     CHECK((j_array.begin() + 3) == it);
+                    CHECK(json::iterator(3 + j_array.begin()) == it);
```

Comment:
Oh stupid me... I didn't see you *added* a line - I thought you *changed* the line to this order...